### ISSUE TITLE
parsing logging

### ISSUE THREAD

**Issue by rharolde** [egberts/bind9_parser#34]
Oddly, it seems to fail on this part:
logging {
        channel "general_file" {
                file "/var/log/named/general.log" versions 10 size 104857600;
                severity dynamic;
                print-time yes;
                print-severity yes;
                print-category yes;
        };
        category "general" {
                "general_file";
                "notice-alert_file";
        };

**Comment by egberts** [comment]
I've added your  sample into [`tests/bug-reports/github-issue-34.named.conf`](https://github.com/egberts/bind9_parser/blob/0.9.98.5/tests/bug-reports/github-issue-34.named.conf).

And got a new output of:
```python
{'logging': {'category_groups': [{'category_group_name': 'general',
                                  'channel_names': ['general_file',
                                                    'notice-alert_file']}],
             'channels': [{'channel_name': 'general_file',
                           'path_name': '/var/log/named/general.log',
                           'print_category': 'yes',
                           'print_severity': 'yes',
                           'print_time': 'yes',
                           'severity': ['dynamic'],
                           'size_spec': [104857600],
                           'versions': 10}]}}
```

**Comment by egberts** [comment]
Fixed in HEAD (4de2cfc75d52ad48d5520549c60c23757d714930)
